baby_squeel.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir.glob('{lib/**/*,*.{md,txt,gemspec}}')
 
-  spec.add_dependency 'activerecord', '>= 6.1', '< 7.1'
+  spec.add_dependency 'activerecord', '>= 6.1', '< 7.2'
```

Comment:
Hey @owst, I hope you don't mind, but I set this back to 6.1. Was there a reason you bumped the minimum version to 6.1.6?

Reply:
Hey @rzane no worries, please modify as you see fit - it was because I thought I read that `ransack >= 4.11` (required for Rails 7.1 support) had a lower-bound of `6.1.6` for `activerecord`, but it seems that it actually [requires `>= 6.1.5`](https://rubygems.org/gems/ransack/versions/4.1.0), so I guess `baby_squeel` should match that lower bound?

Reply:
Hey @rzane any chance you could take another look at this? What do you think of matching Ransack's `activerecord` lower bound?

Reply:
Makes sense to me! Thank you for the explanation!